ConveyorBot/shell_tower/shell_tower_assembly.md

```diff
@@ -70,7 +67,6 @@ What tools will be needed to assemble the Breadcrumb together:
  - small screwdriver (for M2 camera bolts)
  - big screwdriver (for mounting emergency switch)
  - M3 Allen key
- - M5 Allen key
  - M6 Allen key 
  - Paper tape
```

Comment:
TEST REVIEW

Reply:
Yes, but. sgfasdfasd.
Sure ill fix this